protos/local/activity_task.proto

```diff
@@ -72,6 +72,8 @@ enum ActivityCancelReason {
     NOT_FOUND = 0;
     /// Activity was explicitly cancelled
     CANCELLED = 1;
+    /// Activity timed out
+    TIMED_OUT = 2;
 }
 
 
```

Comment:
Mostly unrelated and pedantic, but these some files have 0 EOF blank lines, some 1, some 2.